sdk/src/azure/core/az_mqtt5_rpc_server.c

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <azure/core/az_mqtt5_rpc_server.h>
+#include <azure/core/az_result.h>
+#include <azure/core/internal/az_log_internal.h>
+#include <stdlib.h>
+
+#include <azure/core/_az_cfg.h>
+
+AZ_NODISCARD az_mqtt5_rpc_server_options az_mqtt5_rpc_server_options_default()
+{
+  return (az_mqtt5_rpc_server_options){
+    .subscribe_timeout_in_seconds = AZ_MQTT5_RPC_SERVER_DEFAULT_TIMEOUT_SECONDS,
+  };
+}
+
+AZ_NODISCARD az_result
+az_rpc_server_get_subscription_topic(az_mqtt5_rpc_server* client, az_span out_subscription_topic)
```

Comment:
Per this comment, we shouldn't have az_spans as pointers. We could pass the length out so the caller can slice the out span, but we don't need to for the subscription - let me know if this would be a better practice though!
https://github.com/Azure/azure-sdk-for-c/pull/2628#discussion_r1304952720